theme/less/_content.less

```diff
@@ -18,45 +18,47 @@ article p:first-of-type img
 
 article p
 {
-  margin: 0.5em 0 0 0;
+  margin: 14px 0 0 0;
```

Comment:
TODO: Use EMs instead for consistency.

Reply:
It's not only consistency issue. Do we want margins, paddings and line-heights to be scaled with `<html>` font-size? If yes, we can use here and in other places EMs. What you think?

Reply:
That's a good question, usually I used Pixel for margins and paddings, but EMs or REMs on font-sizes for simplicity in calculation on other projects. I think in general It make sense that the margins are also adjusted accordingly, but sometimes it might make sense to keep in simple and use PX to avoid unexpected behavior with side margins for example.

Short: Both will work, but let's make other margins and paddings following same pattern(either use PX or EMs).

Reply:
OK, to avoid unexpected layouts I'll use Pixels for margins and paddings. In other cases I'll use REM and EMs.